dragonfly/item/compass.go

```diff
@@ -0,0 +1,9 @@
+package item
+
+// Compass is helps to find the spawn place.
```

Comment:
Grammar isn't perfect here, something like "Compass is an item used to find the spawn position." would be better.